### ISSUE TITLE
humanise the reference to someone using the contact form

### ISSUE THREAD

**Issue by katska** [openaustralia/planningalerts#1720]
I love being able to click on someone's name if they were logged in when they make a support request via the form now. It's very functional from an admin point of view. Example below.

User was not logged in when filling out the contact form.

The following user matches the given email address: Firstname Lastname

However, when I'm replying to such emails I feel self conscious about the format of that footer that's in their email coming in, I find myself wanting to remove it before I reply, like it's something I don't want them to have to see.

This issue is to flag that there's a problem, and remind me to look at what the problem might be and ways to improve it.

**Comment by katska** [comment]
yes still needs a review. I welcome comments or for @JoannaHill or @mlandauer  to have a go at improving this if you happen to be touching it for other reasons and you have an idea about how to make it better.

**Comment by mlandauer** [comment]
> I love being able to click on someone's name if they were logged in when they make a support request via the form now. It's very functional from an admin point of view. Example below.
> 
> User was not logged in when filling out the contact form.
> 
> The following user matches the given email address: Firstname Lastname

The example you give here is actually for when someone is *not* logged in but the email address they enter matches an existing user. So specifically they've given a valid email address but in theory they could be impersonating someone so we shouldn't totally believe that they are the user they might be,

In the case where they are logged in it says:

User was logged in when filling out the contact form

Sent by user Jane Smith

**Comment by katska** [comment]
@mlandauer thanks for putting the record straight there. Whether they are signed in or not we can usually match Firstname and Lastname so long as the email address is in the system. While I love the functionality I don't love the language.

The only time I want to see that is in the movie Tron, we believe in the User! 
How about 
"It looks like they weren't logged in when they filled out the contact form. We found a match for the email address provided: Firstname Lastname."

or even just "Hurrah! 🎉  We found a match for the email address provided: Firstname Lastname." use of emoji nice visual clue for admin on scanning the email that there's a match. whaddyou think?
